portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase

```diff
@@ -1173,6 +1173,22 @@ definition {
 		}
 	}
 
+	@description = "#Undefined is appended to the URL when there is an Asset Publisher in a page."
```

Comment:
@description = "This is a test for LPS-128196. View the URL when there is an Asset Publisher in a page."